memory/player_party_manager.py

```diff
@@ -19,35 +19,37 @@ def __init__(self):
         self.base = None
         self.fields_base = None
         self.position = Vec3(None, None, None)
+        self.gameobject_position = Vec3(None, None, None)
         self.leader = None
         self.movement_state = PlayerMovementState.NONE
         self.leader_character = PlayerPartyCharacter.NONE
 
     def update(self):
         if self.memory.ready_for_updates:
-            try:
-                if self.base is None or self.fields_base is None:
-                    singleton_ptr = self.memory.get_singleton_by_class_name(
-                        "PlayerPartyManager"
-                    )
-                    if singleton_ptr is None:
-                        return
-
-                    self.base = self.memory.get_class_base(singleton_ptr)
-                    if self.base == 0x0:
-                        return
-
-                    self.fields_base = self.memory.get_class_fields_base(singleton_ptr)
-
-                else:
-                    # Update fields
-                    self.leader = self.memory.get_field(self.fields_base, "leader")
-                    self._read_position()
-                    self._read_movement_state()
-                    self._read_leader_character()
-            except Exception:
-                # logger.debug(f"PlayerPartyManager Reloading {type(_e)}")
-                self.__init__()
+            # try:
```

Comment:
Is commenting these out intentional? Could it cause problems doing so?

Reply:
i have a bad habit of not putting these back 
they're simply there to prevent the tas from crashing if theres an issue, memory addresses sometimes can not exist for various reasons and this just makes sure it can restore itself - downside is it just hides potential bugs from the end-user